adapter-neo4j/src/main/java/io/nosqlbench/adapter/neo4j/Neo4JAdapterUtils.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 nosqlbench
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.nosqlbench.adapter.neo4j;
+
+import org.apache.commons.lang3.StringUtils;
+
+import org.neo4j.driver.Record;
+import org.neo4j.driver.exceptions.ClientException;
+
+import java.util.NoSuchElementException;
+
+
+public class Neo4JAdapterUtils {
+
+    /**
+     * Mask the digits in the given string with '*'
+     *
+     * @param unmasked The string to mask
+     * @return The masked string
+     */
+    protected static String maskDigits(String unmasked) {
```

Comment:
#nit I think this would be a good candidate to be extracted into some common NB utility so that all drivers can utilize it.